src/main/resources/fabric.mod.json

```diff
@@ -22,6 +22,6 @@
   ],
   "depends": {
     "fabricloader": "*",
-    "minecraft": "1.16.2"
+    "minecraft": "1.16.*"
```

Comment:
Don't make it "all 1.16" as this mod will break in 1.16 and 1.16.1. Please tag it for 1.16.2 to 1.16.* (we don't know if we will get another patch)

Reply:
@rdvdev2  done!